packages/types/src/sdk/events/event.ts

```diff
@@ -150,6 +150,15 @@ export enum Event {
   TENANT_BACKFILL_FAILED = "tenant:backfill:failed",
   INSTALLATION_BACKFILL_SUCCEEDED = "installation:backfill:succeeded",
   INSTALLATION_BACKFILL_FAILED = "installation:backfill:failed",
+
+  // USER
+  USER_GROUP_CREATED = "user_group:created",
+  USER_GROUP_UPDATED = "user_group:updated",
+  USER_GROUP_DELETED = "user_group:deleted",
+  USER_GROUP_USER_ADDED = "user_group_user:added",
```

Comment:
I think the second `user` bit should be on the other side here. For existing events it generally is `entity:action`. So in this case `user_group:user_added`. Same for the events below